repos/kaleido/py/kaleido/scopes/mermaid.py

```diff
@@ -13,30 +13,43 @@ class MermaidScope(BaseScope):
     _scope_flags = ("mermaidjs", "mermaid_config")
     _scope_chromium_args = ("--no-sandbox",)
 
-    def __init__(self, mermaidjs=None, mermaid_config=None, **kwargs):
+   
+    def __init__(self, mermaidjs=None, mermaid_config=None, diagram_config=None, **kwargs):
+        """
+        Constructor of MermaidScope.
+
+        :param mermaidjs: Path to mermaidjs javascript module.
+        :param mermaid_config: Initial config of mermaid javascript object.
+        :param diagram_config: Diagram style config. 
+            See https://mermaid.js.org/config/schema-docs/config.html for more
+            If not specified, will default to the default mermaid styling configuration.
+        """
 
         self._mermaidjs = mermaidjs 
 
         self.default_format = "svg"
         self.default_width = 700
         self.default_height = 500
         self.default_scale = 1
-        self.default_mermaid_config = {"startOnLoad": False, "securityLevel": "loose"}
+        self.default_mermaid_config = {"startOnLoad": False}
+        self.default_diagram_config = {}
 
-        self._initialize_mermaid_config(mermaid_config)
+        self._mermaid_config = self._initialize_config(mermaid_config, self.default_mermaid_config)
```

Comment:
I don't think we need both of these configs, because I think the way it's implemented in render.js, diagram_config will always just override the mermaid_config, we can just pass the default config object to the init